test/parse/test_entity_declaration.rb

```diff
@@ -18,25 +22,22 @@ def parse(internal_subset)
       REXML::Document.new(xml(internal_subset)).doctype
     end
 
+    public
     def test_empty
-      exception = assert_raise(REXML::ParseException) do
+      assert_raise(REXML::ParseException) do
         parse(<<-INTERNAL_SUBSET)
 <!ENTITY>
         INTERNAL_SUBSET
       end
-      assert_equal(<<-DETAIL.chomp, exception.to_s)
-Malformed notation declaration: name is missing
-Line: 5
-Position: 72
-Last 80 unconsumed characters:
- <!ENTITY>  ]> <r/>
-      DETAIL
     end
 
     def test_linear_performance_gt
       seq = [10000, 50000, 100000, 150000, 200000]
       assert_linear_performance(seq, rehearsal: 10) do |n|
-        REXML::Document.new('<!DOCTYPE rubynet [<!ENTITY rbconfig.ruby_version "' + '>' * n + '">')
+        begin
+          REXML::Document.new('<!DOCTYPE rubynet [<!ENTITY rbconfig.ruby_version "' + '>' * n + '">')
```

Comment:
Can we use a valid XML to reproduce this?

Reply:
I fixed with https://github.com/ruby/rexml/pull/182/commits/36ba79eb0fd01c68b37509772d922a80a0f9d530